src/documentation/components/ComponentExample.vue

```diff
@@ -1,32 +1,32 @@
 <template>
   <div class="example-wrapper">
-    <h4 v-if="titleSize === 'h4'" class="-anchor" :id="id">
+    <h4 v-if="titleSize === 'h4' && !hideTitle" class="-anchor" :id="id">
```

Comment:
I would prefer positive checks. It's described in clean code rules of [this doc](https://github.com/academind/clean-code-course-code/blob/general-resources/Clean%20Code%20Summary.pdf):
<img width="1207" alt="Screenshot 2023-03-13 at 12 17 55" src="https://user-images.githubusercontent.com/125541591/224686962-e61fb9aa-73e0-4a67-a4d1-de452eb2b1f7.png">